benches/benches.rs

```diff
@@ -41,7 +37,7 @@ fn policy() -> Result<Policy, Error> {
 }
 
 /// Generate encrypted header with some metadata
-fn generate_encrypted_header(public_key: &PublicKey) -> EncryptedHeader<Aes256GcmCrypto> {
+fn generate_encrypted_header(cc: &CC, public_key: &PublicKey) -> EncryptedHeader {
```

Comment:
about `cc: &CC`. I understand you don't want long names, but here we don't know what is it about.
Is `cc` variable a key? a scheme? Can we make the name of variable and/or type more explicit please.

Clearly I'm also not a big fan of this kind of renaming, which seems ok if you read this code often, but for others it's always a pain. `CoverCryptX25519Aes256` is not that long, considering you don't use it often.
@grydz maybe we need guidelines here

Reply:
I removed all `cc`/`CC`